sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs

```diff
@@ -2932,7 +2932,7 @@ internal virtual void SetNameFieldsIfNull()
                             leaseId: conditions?.LeaseId,
                             owner: owner,
                             group: group,
-                            permissions: permissions.ToSymbolicPermissions(),
+                            permissions: permissions?.ToSymbolicPermissions(),
```

Comment:
I'm not sure I can write unit tests for this, I don't know how to make or get a valid `Owner` or `Group`.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/951b53120fac72ded075240a696fd7c6eedf995d/sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs#L1330-L1331

Reply:
Nice, added a test.